README.md

```diff
@@ -561,6 +561,10 @@ The `check.py` script supports some options:
    tool cannot be found, and you'll see a warning.
  * We have tests from upstream in `tests/spec`, in git submodules. Running
    `./check.py` should update those.
+   
+Note that we are trying to gradually port the legacy wasm-opt tests to use `lit` and `filecheck` as we modify them.
+For passes tests that output wast, this can be done automatically with scripts/port_passes_tests_to_lit.py and for non-passes tests that output wast, see 
```

Comment:
```suggestion
For `passes` tests that output wast, this can be done automatically with `scripts/port_passes_tests_to_lit.py` and for non-`passes` tests that output wast, see 
```